src/test/java/io/devnoob/marble/api/MarbleControllerTest.java

```diff
@@ -181,37 +194,88 @@ void getLatestMarblesByUserIdShouldReturnOkWhenUserExist() throws Exception {
         }
 
         @Test
-        @DisplayName("Should return OK and matchable number when user exist")
-        void getLatestMarblesByUserIdShouldReturnOkAndMatchableNumberWhenUserExist() throws Exception {
+        @DisplayName("Should return marbles in descending order by creation time")
+        void getLatestMarblesByUserIdShouldReturnCorrectResponseBodyAndInFormOfDESC() throws Exception {
+            User user = new User(2L, "test_user2");
+            Mockito.when(userRepository.find(2L)).thenReturn(user);
+            Marble marble2 = new Marble(2L, "test_marble2", 2L, new Timestamp(1623927480), "marble2_test", "story_marble2");
+            Marble marble3 = new Marble(3L, "test_marble3", 2L, new Timestamp(1623927481), "marble3_test", "story_marble3");
             List<Marble> marbles = new LinkedList<>();
-            marbles.add(new Marble(1L, "test_marble1", 1L, new Timestamp(1623917398), "marble1_test", "story_marble1"));
-            marbles.add(new Marble(1L, "test_marble2", 1L, new Timestamp(1623917377), "marble2_test", "story_marble2"));
-            Mockito.when(marbleRepository.getLatestMarblesByUserId(1L, 2))
+            
+            marbles.add(marble3);
+            marbles.add(marble2);
+            Mockito.when(marbleRepository.getLatestMarblesByUserId(2L, 2))
                 .thenReturn(marbles);
 
-            String url = "/api/marble/latest/1?limit=2";
+            String url = "/api/marble/latest/2?limit=2";
```

Comment:
This test case should not rely on the "the method should return the maximum number of marbles when the limit exceeds the total number" feature since the feature should also be tested.